doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -0,0 +1,216 @@
+#. Add the network coordinator application:
+
+   a. In the Welcome page or the Welcome panel, click :guilabel:`Create a new application from sample`.
+   #. Select the :guilabel:`Freestanding` application type, which uses the preinstalled version of the |NCS|.
+   #. Enter a custom application location and name.
+      For example, ``Zigbee_coordinator``.
+   #. Keep the default settings for :guilabel:`nRF Connect SDK` and :guilabel:`nRF Connect Toolchain`.
+   #. In the :guilabel:`Application template`, click :guilabel:`Browse` and search for the Zigbee network coordinator sample.
+      The path for the application template is :file:`nrf/samples/zigbee/network_coordinator`.
+   #. Click :guilabel:`Select` and then click :guilabel:`Create Application`.
+   #. If needed, click :guilabel:`Yes` when prompted if you trust the authors of the files.
+
+#. Generate the network coordinator application:
+
+   a. In the Applications panel, hover with your mouse cursor over the application name and click :guilabel:`Add Build Configuration` button that appears next to it.
+   #. In the configuration options on the :guilabel:`Generate Configuration` screen, select the board onto which you want to program the sample.
+      See the `Requirements`_ section for the list of available build target names.
+      For the purpose of this quick start guide, leave the default values for other options.
+   #. Click :guilabel:`Generate Config`.
+      When the process starts, a Zigbee Coordinator panel appears under the Applications panel.
+      The generation process takes a while and you can observe its progress in the :guilabel:`nRF Connect` terminal.
+
+#. Program the application to the board:
+
+   a. Connect your development kit using the serial port.
+   #. In the Connected Devices panel, click :guilabel:`Refresh Connected Devices`.
+      Your development kit appears on the list.
+   #. In the Actions panel, click :guilabel:`Flash`.
+
+   When the programming is done, the **LED 3** on the development kit turns on to indicate that the Zigbee network is open.
+   After some time, it turns off and the Zigbee network needs to be reopened.
+#. Add the light switch application.
+   Repeat steps 2 to 4 using a custom application location and name, for example ``Zigbee_light_switch``.
+   The path for the application template is :file:`nrf/samples/zigbee/light_switch`.
+#. Add the light bulb application.
+   Repeat steps 2 to 4 using a custom application location and name, for example ``Zigbee_light_bulb``.
+   The path for the application template is :file:`nrf/samples/zigbee/light_bulb`.
+#. Press **Button 1** on the development kit programmed with the network coordinator sample to reopen the Zigbee network.
+   After some time, the devices join the Zigbee network.
+   On the development kit programmed with the light bulb sample, **LED 3** and **LED 4** turn on when the light bulb joins the network.
```

Comment:
LED 4 turns on immediately after application is started, only LED3 is turned on when devices joined the network.